demo-app/src/stories/GenreSelect.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import GenreSelect from '../components/genre-select/GenreSelect';
+
+export default {
+    title: 'GenreSelect',
+    component: GenreSelect
+}
+
+// export const CustomSelectedGenre = () => (
+//     <GenreSelect
+//         genres={['ALL', 'DOCUMENTARY', 'COMEDY', 'HORROR', 'CRIME']}
+//         selectedGenre="COMEDY"
+//         onSelect={(genre) => {
+//             console.log(`Selected genre: ${genre}`);
+//         }}
+//     />);
+
+export const Basic = (args) => <GenreSelect {...args} />;
+Basic.args = {
+    genres:  ['ALL', 'DOCUMENTARY', 'COMEDY', 'HORROR', 'CRIME'],
+    selectedGenre: "COMEDY",
+    onSelect: genre => {
+        console.log(`Selected genre: ${genre}`);
+    }
+}
+
```

Comment:
Here we must to delete onSelect from args and add useState in your Basic component. As result we will have component which can be changed by click and by Control panel + refresh page.
```suggestion
export const Basic = (args) => {
    const [selectedGenre, setSelectedGenre] = useState(args.selectedGenre)

    return <GenreSelect {...args}  selectedGenre={selectedGenre} onSelect={setSelectedGenre}/>
}

Basic.args = {
    genres:  ['ALL', 'DOCUMENTARY', 'COMEDY', 'HORROR', 'CRIME'],
    selectedGenre: "COMEDY",
}
```